server/src/lib/utils/validator.ts

```diff
@@ -65,7 +65,22 @@ function validateBody(body: string, errors: Array<ajv.ErrorObject>) {
   return errors
 }
 
+function isValidSlug(slug: string) {
+  let slugRegex = /^[A-Za-z0-9]+(?:-[A-Za-z0-9]+)*$/;
```

Comment:
Check if some library already has this feature, refer to some typescript repos in Quintype, we might be using some library

Reply:
We can go with regexp for now

Reply:
Searched, I could find only “slugify” function and checks on, if slug has got value.